dasf/transforms/base.py

```diff
@@ -18,8 +18,13 @@
 from dasf.utils.types import is_dask_gpu_dataframe
 from dasf.utils.funcs import block_chunk_reduce
 
+class UUIDObject:
+    def get_uuid(self):
```

Comment:
I like the idea, but maybe we should use something like PipelineObject that we can reuse the structure maybe.

Reply:
I'm gonna merge this PR and make modifications based on some discussions we had previously.